starlark/src/values/int.rs

```diff
@@ -58,6 +58,9 @@ from_int!(u16, i64);
 from_int!(u32, i64);
 // TODO: check for overflow
 from_int!(u64, i64);
+// TODO: check for overflow
```

Comment:
What is the relation of this change with the current PR?

Reply:
Used in this snipped and couple others:

```
impl Inspectable for usize {
    fn inspect(&self) -> Value {
        Value::from(*self)
    }
}
```